content/en/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -74,10 +74,10 @@ used for provisioning VolumeSnapshots. This field must be specified.
 
 ### DeletionPolicy
 
-Volume snapshot classes have a deletionPolicy. It enables you to configure what
-happens to a VolumeSnapshotContent when the VolumeSnapshot object it is bound to
-is to be deleted. The deletionPolicy of a volume snapshot class can either be
-`Retain` or `Delete`. This field must be specified.
+Volume snapshot classes have a [deletionPolicy](/docs/concepts/storage/volume-snapshots/ #delete).
```

Comment:
<img width="796" alt="image" src="https://github.com/kubernetes/website/assets/21279125/7bfef079-521a-4649-b722-477192ba99eb">

Please make sure there is no space.
```suggestion
Volume snapshot classes have a [deletionPolicy](/docs/concepts/storage/volume-snapshots/#delete).
```

Reply:
Thanks @divya-mohan0209 for taking look at this, i will update this.

Reply:
addressed the comment, PTAL!

Reply:
@divya-mohan0209 
PTAL!